### PR TITLE
Backport Froglights + Additional Froglights

### PR DESCRIPTION
**Pull request by GirixK**
Adding the vanilla Pearlescent, Ocher, and Verdant froglights

As well as adding 6 new ones, based on the following materials, in no particular order:
Osmium, Tungstensteel, Iron, Titanium, Aluminium, and Grisium.

image

These blocks offer several slight variations based on various metals and alloys
They fill the purpose of "natural" feeling full-block light sources

I could add more in the future, but for now I didn't want to overdo it, as most of these didn't exactly have a great deal effort put into them, I mostly just felt there was a gap that could be filled using this block.
I might revisit and re-do some of the less clear textures in the future, but for the time being, I'm personally satisfied.

### CODE REVIEW

File: src/main/resources/assets/chisel/lang/en_US.lang [hunk 1]
```diff
@@ -1878,3 +1878,15 @@ tile.glotek.12.desc=Cranberry
...
+#Froglight
+tile.chisel.froglight.name=Froglight
+tile.froglight.0.desc=Pearlescent
+tile.froglight.1.desc=Ocher
+tile.froglight.2.desc=Verdant
+tile.froglight.3.desc=Osmium
+tile.froglight.4.desc=Wolframsteel
+tile.froglight.5.desc=Ferric
+tile.froglight.6.desc=Titanic
+tile.froglight.7.desc=Alumium
```
Comment (glowredman) [inline src/main/resources/assets/chisel/lang/en_US.lang:1891]: Typo: "Alumium"

**Review by glowredman — CHANGES_REQUESTED** [review]
Are the first three textures the same as in modern MC? If so, we can't include them in the mod.

**Comment by Caedis** [comment]
How does EFR get around the reuse of modern resources? Is it because EFR is LGPL while this is only GPL?  

https://github.com/Roadhog360/Et-Futurum-Requiem/tree/master/src/main/resources/assets/minecraft/textures

**Comment by EnderProyects** [comment]
Its ok if you modify some pixels so its not the original texture?.., or how it works the license of textures?

**Comment by glowredman** [comment]
> How does EFR get around the reuse of modern resources? Is it because EFR is LGPL while this is only GPL?
> 
> https://github.com/Roadhog360/Et-Futurum-Requiem/tree/master/src/main/resources/assets/minecraft/textures

See https://www.minecraft.net/en-us/usage-guidelines: "Do **not** redistribute our games or any alterations of our games or game files". IANAL but to me that sounds like "do your own textures". Alterations of official textures *may* count as fair-use but again, IANAL.

File: src/main/resources/assets/chisel/lang/en_US.lang (hunk 1 above)
Comment (GirixK) [reply src/main/resources/assets/chisel/lang/en_US.lang:1891]: That was intentional actually, I already felt like i cheaped out on osmium by not giving it a fancy name, and "[Alumium](https://en.wiktionary.org/wiki/alumium)" apparently used to be a way to refer to Alumin(i)um

**Comment by GirixK** [comment]
@glowredman I changed the bottom-left lamp to have a different texture, do you think this is different enough to fall under fair use? If so, I can polish it up a bit more and change the rest
image

**Comment by GirixK** [comment]
Forgot to actually include an image of the new textures
image

**Comment by Dream-Master** [comment]
@GirixK are you still working on this ? Feature freeze is over

**Comment by GirixK** [comment]
> @GirixK are you still working on this ? Feature freeze is over

As far as I'm concerned, I believe it's done, I feel like the currently colors are varied enough to fit plenty of themes and sceneries

Only things I'm not sure about is incrementing the version number, as well as potentially a cheaper assembler recipe, but I don't actually know how to properly do the assembler recipe

**Comment by Dream-Master** [comment]
the recipes can be easy added in the core mod. But i need to know what about the graphic licens? We need to take care that we not get into copyright issues @GirixK

**Comment by GirixK** [comment]
The first screenshot in this pr is the one which violates the license, the latest image is the (current) assets I've heavily modified and should no longer violate the license, so I'm fairly certain it's free to use 👍

**Comment by Dream-Master** [comment]
> Forgot to actually include an image of the new textures image

so you mean this @GirixK ?

**Comment by Dream-Master** [comment]
> Adding the vanilla Pearlescent, Ocher, and Verdant froglights
> 
> As well as adding 6 new ones, based on the following materials, in no particular order: Osmium, Tungstensteel, Iron, Titanium, Aluminium, and Grisium.
> 
> image
> 
> These blocks offer several slight variations based on various metals and alloys They fill the purpose of "natural" feeling full-block light sources
> 
> I could add more in the future, but for now I didn't want to overdo it, as most of these didn't exactly have a great deal effort put into them, I mostly just felt there was a gap that could be filled using this block. I might revisit and re-do some of the less clear textures in the future, but for the time being, I'm personally satisfied.

This are the original ones @GirixK  ?

**Comment by Dream-Master** [comment]
if so what do you think ? @glowredman @mitchej123 @boubou19

**Comment by GirixK** [comment]
Yes, the 3x3 is mojang assets
The 2x5 with the signs are my assets, which I heavily modified

**Review by glowredman — APPROVED** [review]
lgtm

**Pull request merged**